x/chainlink/types/msg.go

```diff
@@ -56,3 +57,47 @@ func (m *MsgFeedData) ValidateBasic() error {
 	}
 	return nil
 }
+
+var _ sdk.Msg = &ModuleOwner{}
+
+func NewModuleOwner(address sdk.Address, pubKey []byte) *ModuleOwner {
+	return &ModuleOwner{
+		Address: address.Bytes(),
+		PubKey:  pubKey,
+	}
+}
+
+func (m *ModuleOwner) Route() string {
+	return RouterKey
+}
+
+func (m *ModuleOwner) Type() string {
+	return AddModuleOwner
+}
+
+func (m *ModuleOwner) ValidateBasic() error {
+	// TODO: add proper cosmos address and pubkey validation
+	return nil
+}
+
+func (m *ModuleOwner) GetSignBytes() []byte {
+	bz := ModuleCdc.MustMarshalJSON(m)
+	return sdk.MustSortJSON(bz)
+}
+
+func (m *ModuleOwner) GetSigners() []sdk.AccAddress {
```

Comment:
would there ever be the case where there will be more than one `m.Address` set in a `ModuleOwner` obj?